manifests/params.pp

```diff
@@ -333,6 +349,10 @@
       $agent_restart_command = undef
       $unavailable_runmodes = ['cron', 'systemd.timer']
     }
+    'Archlinux': {
+      $agent_restart_command = "/usr/bin/systemctl reload-or-restart ${service_name}"
+      $unavailable_runmodes = ['cron']
```

Comment:
There is no cron available in Arch in a clean install - https://wiki.archlinux.org/index.php/Cron#Installation

As such, even an absent resource will fail because there's no provider to execute it.